experimental/PropertyDDS/services/property-query-service/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@fluid-experimental/property-query-service",
   "version": "0.59.3000",
```

Comment:
I don't know anything about this package - is it ok to publish it?

Reply:
Good question - I'll instead convert it to `@fluid-internal` perhaps